classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
+		$this->columns[] = Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value) AS section_abbrev');
+
+		$q->groupBy(Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value)'));
+		$q->groupBy(Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value)'));
+
+		$q->leftJoin('section_settings as stpl', function($join) use($primaryLocale) {
+			$join->on('s.section_id', '=', Capsule::raw('`stpl`.`section_id`'));
```

Comment:
@kaschioudi and @asmecher I'm running into some problems with this line and 63, 69 and 75 below. When written like the following, it passes `mysql` tests but fails `postgres` tests:

```
'`stpl`.`section_id`'
```

Same with:

```
"`stpl`.`section_id`"
```

I've tried other things which pass `postgres` but cause problems with `mysql`. I've tried:

```
'"stpl"."section_id"'
```
And
```
"'stpl'.'section_id'"
```
And
```
'stpl.section_id'
```
Do either of you have an idea for what the syntax should be for this to pass muster with `postgres` and `mysql`?

Reply:
Looking into this now...

Reply:
I don't think any escaping is needed here. This works for me with PostgreSQL: https://github.com/asmecher/ojs/commit/9ca1c8bf65ffedf4fc0282d7fcbd73f01602107b